liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/ScaffoldDTO.kt

```diff
@@ -0,0 +1,62 @@
+package org.phoenixframework.liveview.data.dto
+
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import org.phoenixframework.liveview.domain.base.ComposableBuilder
+import org.phoenixframework.liveview.domain.base.ComposableView
+import org.phoenixframework.liveview.domain.extensions.toColor
+
+class ScaffoldDTO private constructor(builder: Builder) :
+    ComposableView(modifier = builder.modifier) {
+    // set by LiveViewCoordinator.extractChildren
+    var topAppBar: TopAppBarDTO? = null
+    private val backgroundColor: Color = builder.backgroundColor
+
+    @Composable
+    fun Compose(content: @Composable () -> Unit) {
+        Scaffold(modifier = modifier, backgroundColor = backgroundColor, topBar = {
+            topAppBar?.Compose()
+        }) { paddingValues ->
+            paddingValues
```

Comment:
This is marked as unused.  Should the `paddingValues` be used in the `content` or elsewhere or should it be marked as unused?

Reply:
padding values would be used in content

Reply:
@Kashif-E can you check if https://github.com/liveviewnative/liveview-client-jetpack/pull/53/commits/ad834fe884fab842107bb7fc81367dfed031a56b is the correct way to pass the `paddingValues` to all scaffold root content elements and only the root content elements?  I went purely on the docs, but you can double check how it actually works to ensure it is correct.

Reply:
yes @KronicDeth , its the right way